test/e2e/networkpolicy_test.go

```diff
@@ -35,20 +35,56 @@ import (
 	"antrea.io/antrea/pkg/features"
 )
 
-func skipIfNetworkPolicyStatsDisabled(tb testing.TB, data *TestData) {
-	skipIfFeatureDisabled(tb, data, features.NetworkPolicyStats, true, true)
+func skipIfNetworkPolicyStatsDisabled(tb testing.TB) {
+	skipIfFeatureDisabled(tb, features.NetworkPolicyStats, true, true)
 }
 
-func TestNetworkPolicyStats(t *testing.T) {
-	skipIfNotIPv4Cluster(t)
+// TestNetworkPolicy is the top-level test which contains all subtests for
+// NetworkPolicy related test cases so they can share setup, teardown.
+func TestNetworkPolicy(t *testing.T) {
 	skipIfHasWindowsNodes(t)
 
 	data, err := setupTest(t)
 	if err != nil {
 		t.Fatalf("Error when setting up test: %v", err)
 	}
 	defer teardownTest(t, data)
-	skipIfNetworkPolicyStatsDisabled(t, data)
+
+	t.Run("testNetworkPolicyStats", func(t *testing.T) {
+		skipIfNotIPv4Cluster(t)
+		skipIfNetworkPolicyStatsDisabled(t)
+		testNetworkPolicyStats(t, data)
+	})
+	t.Run("testDifferentNamedPorts", func(t *testing.T) {
+		testDifferentNamedPorts(t, data)
+	})
+	t.Run("testDefaultDenyIngressPolicy", func(t *testing.T) {
+		testDefaultDenyIngressPolicy(t, data)
+	})
+	t.Run("testDefaultDenyEgressPolicy", func(t *testing.T) {
+		testDefaultDenyEgressPolicy(t, data)
+	})
+	t.Run("testNetworkPolicyResyncAfterRestart", func(t *testing.T) {
+		testNetworkPolicyResyncAfterRestart(t, data)
+	})
+	t.Run("testIngressPolicyWithoutPortNumber", func(t *testing.T) {
+		testIngressPolicyWithoutPortNumber(t, data)
+	})
+	t.Run("testIngressPolicyWithEndPort", func(t *testing.T) {
+		testIngressPolicyWithEndPort(t, data)
+	})
+}
+
+func testNetworkPolicyStats(t *testing.T, data *TestData) {
+	cc := []configChange{
+		{"NetworkPolicyStats", "true", true},
+	}
+	ac := []configChange{
+		{"NetworkPolicyStats", "true", true},
+	}
+	if err := data.mutateAntreaConfigMap(cc, ac, true, true); err != nil {
```

Comment:
I think it will revert the change @antoninbas did on purpose 05eee251c9c53400277def576f92d614ca234898

Reply:
ah, thanks to point out this, I thought I fixed all related conflicts, will fix it.

Reply:
done